app/upgrades/v0_7_0/upgrades.go

```diff
@@ -15,13 +22,57 @@ const (
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
+	StoreUpgrades: storetypes.StoreUpgrades{
+		Added: []string{
+			feegrant.StoreKey, feemarkettypes.StoreKey,
+		},
+		Renamed: nil,
+		Deleted: nil,
+	},
 }
 
 func CreateUpgradeHandler(
 	moduleManager *module.Manager,
 	configurator module.Configurator,
+	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx context.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
-		return moduleManager.RunMigrations(ctx, configurator, vm)
+
+		vm, err := moduleManager.RunMigrations(ctx, configurator, vm)
+		if err != nil {
+			return vm, err
+		}
+
+		if err := ConfigureFeeMarketModule(ctx, keepers); err != nil {
+			return vm, err
+		}
```

Comment:
be it within `RunMigrations` or anything within the scope of `CreateUpgradeHandler`, we should have way more logging at every single step of this process. We need to know exactly how our migration is progressing during the migration/upgrade.

Reply:
Agreed! Adding more logs...